src/views/Claiming/constants.ts

```diff
@@ -18,6 +18,7 @@ export const CLAIMING_LIST_COLUMNS = {
 
 export const CLAIMING_LIST_SORTABLE_FIELDS = [
   CLAIMING_LIST_COLUMNS.receiptDate,
+  CLAIMING_LIST_COLUMNS.vendorCode,
```

Comment:
Grouping will be based on `organizationId`, not `vendorCode`, and will be triggered by action button. Please remove it from this array